pytmc/bin/pragmalint.py

```diff
@@ -21,9 +21,12 @@
 
 PRAGMA_START_RE = re.compile('{attribute')
 PRAGMA_RE = re.compile(
-    r"^{\s*attribute[ \t]+'pytmc'[ \t]*:=[ \t]*'([\s\S]*)'}$",
+    r"^{\s*attribute[ \t]+'pytmc'[ \t]*:=[ \t]*'(?P<setting>[\s\S]*)'}$",
     re.MULTILINE
 )
+PRAGMA_LINE_RE = re.compile(r"([^;'\n]+)", re.MULTILINE)
+PRAGMA_SETTING_RE = re.compile(r"^\s*(?P<title>[a-zA-z0-9]+):(?P<setting>.*)$")
```

Comment:
Maybe we could be a bit more lax on whitespace near the "title", and strip the whitespace before/after "setting"?
Something like `r"^\s*(?P<title>[a-zA-z0-9]+)\s*:\s*(?P<setting>\S*)\s*$`

Reply:
Also, your original regexes are here: https://github.com/slaclab/pytmc/blob/6a35be09f5b5e9ab47d14845732b32a370c8e827/pytmc/pragmas.py#L16

Wonder if we should refer to at least one of them here (or fix those up?)

Reply:
Good point. Using the common specification and updating it.  

With regards to the whitespace handling, I think `strip` is a more thorough solution. Some of our settings actually feature spaces inside so the `\S` should break e.g. on `field: ZNAM OFF`. If we still use the `.` inside greedy implementation means that the last `\s` won't be sought and we'll still need to `strip` the string later.

Reply:
```suggestion
PRAGMA_SETTING_RE = re.compile(r"^\s*(?P<title>[a-zA-Z0-9]+)\s*:\s*(?P<setting>.*?)\s$")
```